### PR TITLE
Additional plugin check fixes

### PR DESCRIPTION
**Pull request by brianhogg**
## Description
Additional fixes reported by the plugin checker.

## How has this been tested?
Manually.

## Checklist:
- [ ] This PR requires and contains at least one changelog file. 
- [ ] My code has been tested.
- [ ] My code passes all existing automated tests. 
- [ ] My code follows the LifterLMS Coding & Documentation Standards.

### CODE REVIEW

**Comment by brianhogg** [comment]
No changelog since dev has one from `fix/plugin-checks` already.

File: includes/notifications/views/class.llms.notification.view.quiz.graded.php [hunk 1]
```diff
@@ -80,13 +80,13 @@ protected function set_body_email() {
 		);
 
 		ob_start();
+		// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- Output is escaped by the mailer.
 		echo $mailer->get_table_html( $rows );
```
Comment (ideadude) [inline includes/notifications/views/class.llms.notification.view.quiz.graded.php]:
```suggestion
		$mailer->output_table_html( $rows );
```

File: includes/notifications/views/class.llms.notification.view.manual.payment.due.php [hunk 2]
```diff
@@ -83,12 +82,15 @@ private function set_body_email() {
...
-		<h4><?php printf( __( 'Order #%s', 'lifterlms' ), '{{ORDER_ID}}' ); ?></h4>
-		<?php echo $mailer->get_table_html( $rows ); ?>
-		<p><a href="{{ORDER_URL}}"><?php _e( 'Pay Invoice', 'lifterlms' ); ?></a></p>
+		?><p><?php printf( esc_html__( 'Hello %s,', 'lifterlms' ), '{{CUSTOMER_NAME}}' ); ?></p>
+		<p><?php printf( esc_html__( 'A payment for your subscription to %1$s is due.', 'lifterlms' ), '{{PRODUCT_TITLE}}' ); ?></p>
+		<p><?php printf( esc_html__( 'Sign in to your account and %1$spay now%2$s.', 'lifterlms' ), '<a href="{{ORDER_URL}}">', '</a>' ); ?></p>
+		<h4><?php printf( esc_html__( 'Order #%s', 'lifterlms' ), '{{ORDER_ID}}' ); ?></h4>
+		<?php
+		// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- Output is escaped by the mailer.
+		echo $mailer->get_table_html( $rows );
```
Comment (ideadude) [inline includes/notifications/views/class.llms.notification.view.manual.payment.due.php]:
```suggestion
		$mailer->output_table_html( $rows );
```

File: includes/notifications/views/class.llms.notification.view.upcoming.payment.reminder.php [hunk 3]
```diff
@@ -92,23 +91,26 @@ private function set_body_email() {
...
 			// Translators: %s= The order ID.
-			printf( __( 'Order #%s', 'lifterlms' ), '{{ORDER_ID}}' );
+			printf( esc_html__( 'Order #%s', 'lifterlms' ), '{{ORDER_ID}}' );
 		?>
 		</h4>
-		<?php echo $mailer->get_table_html( $rows ); ?>
-		<p><a href="{{ORDER_URL}}"><?php _e( 'Update Payment Method', 'lifterlms' ); ?></a></p>
+		<?php
+		// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- Output is escaped by the mailer.
+		echo $mailer->get_table_html( $rows );
```
Comment (ideadude) [inline includes/notifications/views/class.llms.notification.view.upcoming.payment.reminder.php]:
```suggestion
		$mailer->output_table_html( $rows );
```

File: templates/admin/reporting/tabs/students/achievements.php [hunk 4]
```diff
@@ -20,4 +20,5 @@
 		'student' => $student,
 	)
 );
+// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- Escaped in method.
 echo $table->get_table_html();
```
Comment (ideadude) [inline templates/admin/reporting/tabs/students/achievements.php]:
```suggestion
$table->output_table_html();
```

**Pull request merged**